dotnetv3/Cognito/Scenarios/Cognito_Basics/settings.json

```diff
@@ -0,0 +1,12 @@
+{
+  "parameterGroupName": "test-parameter-group",
+  "parameterGroupFamily": "mysql8.0",
+  "modifyParameterName": "auto_increment_offset",
+  "modifyParameterValue": "3",
+  "engineName": "mysql",
```

Comment:
These settings look odd for Cognito: engine, instance, instanceClass...are they RDS configurations?

Reply:
Not the right settings at all. I had the settings hard-coded so I didn't notice that the settings file wasn't properly implemented. Fixed.